neurokit2/signal/signal_interpolate.py

```diff
@@ -1,24 +1,28 @@
 # -*- coding: utf-8 -*-
 import numpy as np
-import scipy.interpolate
+from scipy.interpolate import interp1d
```

Comment:
@JohnDoenut I prefer the first version as it is IMO always clearer to retain the module.submodule names throughout, even if it seems a bit more verbose. But it's easier to know from where it comes